components/vf-summary/vf-summary.scss

```diff
@@ -12,6 +12,30 @@
     margin-bottom: 0;
   }
 
+  // vf-summary can also sit inside a vf-box - so we need to adjust some things to match
```

Comment:
Quite a few things might sit inside vf-box and i expect we'll have other things that might sit inside other things with background colours. When we have time, would be nice to have a method to support some kind of `vf-give-it-contrast` method.

Reply:
Id rather work on something that either fixes this programmatically or adds specificity than sling yet another class onto something.

Reply:
I think we can go with this now 'as is' and come back to it -- it'd be nice to look at component 'theming' in more detail for the next beta